### PR TITLE
Fix: provide changes according to QA

### PR DESCRIPTION
**Pull request by oyershov**

### CODE REVIEW

**Comment by pkucherenk0** [comment]
I was not able to proceed in testing 
Selection_999(910)

File: src/assets/images/SearchFieldCloseIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    className?: string;
+    onClick?: (e?: any) => void;
+}
+
+export const SearchFieldCloseIcon: React.FC<SearchFieldCloseIconProps> = (props: SearchFieldCloseIconProps) => {
+    return (
+      <svg className={props.className} onClick={props.onClick} width="15" height="15" viewBox="0 0 15 15" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <path d="M7.5 0C11.6475 0 15 3.3525 15 7.5C15 11.6475 11.6475 15 7.5 15C3.3525 15 0 11.6475 0 7.5C0 3.3525 3.3525 0 7.5 0ZM10.1925 3.75L7.5 6.4425L4.8075 3.75L3.75 4.8075L6.4425 7.5L3.75 10.1925L4.8075 11.25L7.5 8.5575L10.1925 11.25L11.25 10.1925L8.5575 7.5L11.25 4.8075L10.1925 3.75Z" fill="#737F92"/>
+      </svg>
+    );
```
Comment (akhlopiachyi) [inline src/assets/images/SearchFieldCloseIcon.tsx:13]: please fix indentation

File: src/assets/images/SearchFieldCloseIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+interface SearchFieldCloseIconProps {
+    className?: string;
+    onClick?: (e?: any) => void;
+}
+
+export const SearchFieldCloseIcon: React.FC<SearchFieldCloseIconProps> = (props: SearchFieldCloseIconProps) => {
+    return (
+      <svg className={props.className} onClick={props.onClick} width="15" height="15" viewBox="0 0 15 15" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <path d="M7.5 0C11.6475 0 15 3.3525 15 7.5C15 11.6475 11.6475 15 7.5 15C3.3525 15 0 11.6475 0 7.5C0 3.3525 3.3525 0 7.5 0ZM10.1925 3.75L7.5 6.4425L4.8075 3.75L3.75 4.8075L6.4425 7.5L3.75 10.1925L4.8075 11.25L7.5 8.5575L10.1925 11.25L11.25 10.1925L8.5575 7.5L11.25 4.8075L10.1925 3.75Z" fill="#737F92"/>
```
Comment (akhlopiachyi) [inline src/assets/images/SearchFieldCloseIcon.tsx:11]: use color env for icon
I guess, it's var(--icons)

File: src/components/OrderInput/index.tsx [hunk 3]
```diff
@@ -82,7 +82,7 @@ export const OrderInput: React.FunctionComponent<OrderInputProps> = React.memo((
         <div className={cx}>
             <fieldset className={fieldsetFocusedClass}>
                 <CustomInput
-                    type="text"
+                    type="number"
```
Comment (josadcha) [inline src/components/OrderInput/index.tsx:85]: image
this what happens when OrderInput has type number

File: src/components/TabPanel/index.tsx [hunk 4]
```diff
@@ -54,6 +55,11 @@ export interface TabPanelProps {
      * Optinal JSX element to head
      */
     optionalHead?: React.ReactNode;
+    /**
+     * Determines whether tab header should looks like dropdown or tab switcher
+     */
+    isMobileDevice?: boolean;
+
```
Comment (josadcha) [inline src/components/TabPanel/index.tsx:62]: remove line please

**Review by josadcha — CHANGES_REQUESTED** [review]

File: src/components/OrderInput/index.tsx (hunk 3 above)
Comment (oyershov) [reply src/components/OrderInput/index.tsx:85]: fixed, thanks👍

**Review by akhlopiachyi — APPROVED** [review]

**Review by josadcha — APPROVED** [review]

**Pull request merged**